modules/effects/spec/effects_error_handler.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { ErrorHandler, Type } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { Action, Store } from '@ngrx/store';
+import { Observable, of } from 'rxjs';
+import {
+  createEffect,
+  EFFECTS_ERROR_HANDLER,
+  EffectsModule,
+  EffectSources,
+} from '..';
+
+describe('NgRx Effects Error Handler spec', () => {
+  let subscriptionCount: number;
+  let globalErrorHandler: jasmine.Spy;
+  let dispatch: jasmine.Spy;
+  let storeNext: jasmine.Spy;
+
+  function makeEffectTestBed(...providers: any[]) {
+    subscriptionCount = 0;
+
+    TestBed.configureTestingModule({
+      imports: [EffectsModule.forRoot([ErrorEffect])],
+      providers: [
+        {
+          provide: Store,
+          useValue: {
+            dispatch: jasmine.createSpy('dispatch'),
+            next: jasmine.createSpy('storeNext'),
+          },
+        },
```

Comment:
I think you can just `provideMockStore()` here

Reply:
I don't believe that I can because I need to later assert on the `Store.next` method being called, which is immediately called when the test bed initialises, not allowing me to use the MockStore's public `scannedActions$` to assert on. I guess I could change the effect under test to only error when I tell it to by controlling a subject that it returns?

Reply:
I attempted to do the above, but it ended up being even more complicated and difficult to read; I think what I have may be the simpler option unless I've missed something obvious.